[UI/Admin/Sign-in] - Modified login UI Form

Chie0430 opened the pull request:
[UI/Admin/Sign-in] - Modified login UI Form
- Modified sellerlogin.blade.php
   1. Changed comment infomation
- Create adminlogin.blade.php

Chie0430 committed:
[UI/Admin/Sign-in] - Modified login UI Form
- Modified sellerlogin.blade.php
   1. Changed comment infomation
- Create adminlogin.blade.php

Ninit29 commented on resources/views/auth/adminlogin.blade.php:
```diff
@@ -0,0 +1,106 @@
+            <div class="">
+                <div class="ms-0">
+                    <a  href="{{ url('/') }}" class="logo-img">
+                        <img src="images/common/Logo.png" alt="logo" style="width: 150px; height:150px;">
```
Please use asset when access images like this below:

https://github.com/kredo-school/18tha-gj_mall/pull/14/files#diff-12e05a6ae0b6a6da563eb899687b9da0d94073554ad4041f40d6cbb2816688e5R32

Chie0430 committed:
[UI/Admin/Sign-in] - Modified login UI Form
- Modified sellerlogin.blade.php
- Modified adminlogin.blade.php
- Modified login.blade.php
- Modified register.blade.php
   1. Changed image tag setting
